xt/lib/PageObject/App/Menu.pm

```diff
@@ -122,9 +122,12 @@ sub click_menu {
             ok($label,"Found label $label");
 
             my $submenu = $item->find("//*[\@id='$label']");
-            ok($submenu,"Submenu found " . $submenu->get_text);
+            ok($submenu,"Submenu found '" . $submenu->get_text . "'");
 
-            $submenu->click;
+            #Click toggle the foldable menus. No need if it is already opened
```

Comment:
"click menu" is the specification and that's what it does. Maybe the intent of the caller is to *close* the submenu. From this code, we can't tell. I'm not sure this change is the correct one. Isn't there a better fix we can apply?

Reply:
We don't have any tests to close sub-menus. This is part of the menu feature test, which validates that each menu is accessible.

Reply:
Even if we don't have any tests depending on the now-removed behaviour, I'm concerned to have a function which doesn't do what it implies.

Reply:
The major problem it that the click, compounded with the retries, can toggles the submenu, opening and closing until the proper timing makes the validation works.